docs/js/contribution.md

```diff
@@ -0,0 +1,173 @@
+---
+id: contribution-model-sap-cloud-sdk-for-javascript-typescript
+title: Contribution model
+sidebar_label: Contribution model
+description:
+keywords:
+- sap
+- cloud
+- sdk
+- cloud native
+- cloud sdk
+- sap cloud sdk
+---
+
+## Introduction
+
+SAP Cloud SDK for JavaScript welcomes contributions that bring value to SDK users by simplifying and/or improving cloud application development within SAP ecosystem. The contributed feature or API library becomes a part of SDK and is released under SAP Cloud SDK umbrella into `@sap-cloud-sdk` to the [npmjs.com](https://www.npmjs.com/).
```

Comment:
Minor: I would put a few articles like:  `the cloud application development` and `the SAP Cloud SDK umbrella`. But from my experience germans tend to put too many articles and people with an eastern European language background too little.

Reply:
Good observation, which is indeed true. But I haven't yet run it via grammar checker and one more commit is pending as I'm still editing.